eth/vm/computation.py

```diff
@@ -402,12 +437,14 @@ def generate_child_computation(self, child_msg: Message) -> 'BaseComputation':
                 self.state,
                 child_msg,
                 self.transaction_context,
+                self.tracer
```

Comment:
*nitpick* trailing commas are nice, because adding a new argument only diffs one line instead of two. (like here, line 439 didn't have to change).